package.json

```diff
@@ -65,10 +65,11 @@
     "eslint": "^7.32.0",
     "eslint-plugin-lit": "^1.5.1",
     "focus-visible": "^5.2.0",
-    "husky": "^4.3.8",
+    "husky": "^8.0.0",
     "lodash": "^4.17.21",
     "marked": "^4.0.10",
     "nodemon": "^2.0.12",
+    "node-sass": "^8.0.0",
```

Comment:
This is not how we need to update to dart-sass. Please see this commit for all the details that are needed. 

https://github.com/AlaskaAirlines/auro-badge/pull/45/commits/f1edf77ba148adec56ebbd4fe0e2965ec9465977

By not updating wc-sass-render, you still have a dependency on node-sass. That's why the `node-sass` commands in the npm scripts are still working.